src/ekore/anomalous_dimensions/unpolarized/space_like/as4/fhmruvv_approximations/ggq.py

```diff
@@ -0,0 +1,130 @@
+r"""The unpolarized, space-like anomalous dimension :math:`\gamma_{gq}^{(3)}`."""
+import numba as nb
+
+from ......harmonics import cache as c
+from ......harmonics.log_functions import lm12, lm13, lm14, lm15
+
+
+@nb.njit(cache=True)
+def gamma_gq(n, nf, cache, variation):
+    r"""Compute the |N3LO| gluon-quark singlet anomalous dimension.
+
+    The routine is taken from a private file.
+
+    Parameters
+    ----------
+    n : complex
+        Mellin moment
+    nf : int
+        Number of active flavors
+    cache: numpy.ndarray
+        Harmonic sum cache
+    variation : int
+        |N3LO| anomalous dimension variation
+
+    Returns
+    -------
+    complex
+        |N3LO| gluon-quark singlet anomalous dimension
+        :math:`\gamma_{gq}^{(3)}(N)`
+
+    """
+    S1 = c.get(c.S1, cache, n)
+    S2 = c.get(c.S2, cache, n)
+    S3 = c.get(c.S3, cache, n)
+    S4 = c.get(c.S4, cache, n)
+    S5 = c.get(c.S5, cache, n)
+    nf2 = nf * nf
+
+    # Known large-x coefficients
+    x1L5cff = 1.3443073 * 10 - 5.4869684 * 0.1 * nf
+    x1L4cff = 3.7539831 * 10**2 - 3.4494742 * 10 * nf + 8.7791495 * 0.1 * nf2
+
+    # Small-x, Casimir scaled from P_gg (approx. for bfkl1)
+    bfkl0 = -8.3086173 * 10**3 / 2.25
+    bfkl1 = (-1.0691199 * 10**5 - nf * 9.9638304 * 10**2) / 2.25
+
+    # The resulting part of the function
+    P3GQ01 = (
+        +bfkl0 * (-(6 / (-1 + n) ** 4))
+        + bfkl1 * 2 / (-1 + n) ** 3
+        + x1L4cff * lm14(n, S1, S2, S3, S4)
+        + x1L5cff * lm15(n, S1, S2, S3, S4, S5)
+    )
+
+    # The selected approximations for nf = 3, 4, 5
+    if nf == 3:
+        P3gqApp1 = (
+            P3GQ01
+            + 3.4 * bfkl1 * (-(1 / (-1 + n) ** 2))
+            - 161562.0 * 1 / ((-1 + n) * n)
+            + 36469.0 * 1 / n
+            + 72317.0 * (-(1 / n**2))
+            - 3977.3 * lm12(n, S1, S2)
+            + 484.4 * lm13(n, S1, S2, S3)
+        )
+        P3gqApp2 = (
+            P3GQ01
+            + 5.4 * bfkl1 * (-(1 / (-1 + n) ** 2))
+            - 546482.0 * 1 / ((-1 + n) * n)
+            - 39464.0 * 1 / n
+            - 401000.0 * (-(1 / n**2))
+            + 13270.0 * lm12(n, S1, S2)
+            + 3289.0 * lm13(n, S1, S2, S3)
+        )
+    elif nf == 4:
+        P3gqApp1 = (
+            P3GQ01
+            + 3.4 * bfkl1 * (-(1 / (-1 + n) ** 2))
+            - 158805.0 * 1 / ((-1 + n) * n)
+            + 35098.0 * 1 / n
+            + 87258.0 * (-(1 / n**2))
+            - 4834.1 * lm12(n, S1, S2)
+            + 176.6 * lm13(n, S1, S2, S3)
+        )
+        P3gqApp2 = (
+            P3GQ01
+            # + 5.0*bfkl1* (-(1 / (-1 + n) ** 2))
```

Comment:
```suggestion
            # this was a bug we detected
            # + 5.0*bfkl1* (-(1 / (-1 + n) ** 2))
```
right?

Reply:
I don't think so... In any case the comment is now gone in the online file:
https://arxiv.org/src/2310.05744v1/anc/xPgq3a.f